tools/python/generate_styles_override.py

```diff
@@ -36,8 +36,8 @@ def copy_style_file(style_path, drules_suffix, target_path):
     shutil.rmtree(output_name)
 os.makedirs(output_name)
 
-paths = ['clear/style-clear', 'clear/style-night', 'vehicle/style-clear', 'vehicle/style-night']
-suffixes = ['_clear', '_dark', '_vehicle_clear', '_vehicle_dark']
+paths = ['default/light', 'default/dark', 'vehicle/light', 'vehicle/dark']
+suffixes = ['_default_light', '_default_dark', '_vehicle_light', '_vehicle_dark']
```

Comment:
and here

Reply:
I don't really know what these python tools do. I just changed the styles names. I don't know why there's no outdoors styles in there

Reply:
@pastk it's the question to you )

Reply:
This script is used for designer tool. Since the designer is broken, we can ignore any inconsistency here

Reply:
It would be great to revive designer back.